recipes/vs2008_express_vc_python_patch/meta.yaml

```diff
@@ -0,0 +1,29 @@
+package:
+  name: vs2008_express_vc_python_patch
+  version: 1.0
+
+build:
+  number: 0
+  skip: true  # [not win]
+  script:
+    - if not exist %SCRIPTS% mkdir %SCRIPTS%
+    - if errorlevel 1 exit 1
+    - xcopy /i /v /f %RECIPE_DIR%\\vs2008_patch\\setup_x64.bat %SCRIPTS%\\setup_x64.bat
+    - if errorlevel 1 exit 1
+    - xcopy /i /v /f /s %RECIPE_DIR%\\vs2008_patch\\x64 %SCRIPTS%
```

Comment:
Try changing this to `xcopy /i /v /f /s %RECIPE_DIR%\\vs2008_patch\\x64 %SCRIPTS%\\x64`.